src/redux/configureStore.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
+import categoriesReducer from './categories/categories';
+import booksReducer from './books/books';
+
+export default combineReducers({
+  categoriesReducer,
+  booksReducer,
+});
```

Comment:
- [ ] Please import `configureStore` also from redux to create your redux store.
- [ ] Kindly define your combined reducer like:
```
const reducer = combineReducers({
  categoriesReducer,
  booksReducer,
});
```

- [ ] Please create your store using your `configureStore` like that:
```
export default const store = configureStore(reducer);
```